### ISSUE TITLE
New tab should open after pinned tabs

### ISSUE THREAD

**Issue by blackjackshellac** [ChaosinaCan/ClassicTabs#28]
If opening a tab from a pinned tab, it is inserted next to the current pinned tab.  It would be nice if it would open the new tab adjacent to the last pinned tab, or perhaps add a toggle to skip pinned tabs when inserting.

**Comment by djmarcone** [comment]
yes, this extension is great but it should open the new tab after the pinned.
